actions/templates.go

```diff
@@ -85,7 +84,7 @@ func GetTemplates(c *gin.Context) {
 
 	scopeMap := c.QueryMap("scopes")
 
-	s := templatesvc.NewTemplateService(storage.GetFromContext(c), s3.GetFromContext(c))
+	s := templatesvc.New(storage.GetFromContext(c), s3.GetFromContext(c))
```

Comment:
```suggestion
	service := templatesvc.New(storage.GetFromContext(c), s3.GetFromContext(c))
```

Be consistent in naming of the variables throughout the code.